File: backend/src/main/java/com/bluebee/smartsupply/model/OrderStatus.java
package com.bluebee.smartsupply.model;

import java.io.Serializable;
import java.sql.*;

public class OrderStatus implements Serializable {

    private Integer orderstatusid;
    private Integer orderid;
    private Integer orderstatuscd;
    private Timestamp orderts;
    private Integer updateuid;

    public Integer getUpdateuid() {
        return updateuid;
    }

    public void setUpdateuid(Integer updateuid) {
        this.updateuid = updateuid;
    }

    public Integer getOrderstatusid(){
    return this.orderstatusid;
    }
    public void setOrderstatusid(Integer orderstatusid){ 
    this.orderstatusid=orderstatusid;
    }
    public Integer getOrderid(){ 
    return this.orderid;
    }
    public void setOrderid(Integer orderid){ 
    this.orderid=orderid;
    }
    public Integer getOrderstatuscd(){ 
    return this.orderstatuscd;
    }
    public void setOrderstatuscd(Integer orderstatuscd){ 
    this.orderstatuscd=orderstatuscd;
    }
    public Timestamp getOrderts(){ 
    return this.orderts;
    }
    public void setOrderts(Timestamp orderts){ 
    this.orderts=orderts;
    }

}

File: frontend/app/src/main/java/com/asuravan/smartsupply/pojo/OrderStatus.kt
package com.asuravan.smartsupply.pojo

import android.os.Parcel
import android.os.Parcelable
import java.sql.Timestamp

class OrderStatus() :Parcelable {

   var orderstatusid: Int? = null
   var orderid: Int? = null
   var orderstatuscd: Int? = null
   var orderts: Timestamp? = null
   var updateuid: Int? = null

    constructor(parcel: Parcel) : this() {
        orderstatusid = parcel.readValue(Int::class.java.classLoader) as? Int
        orderid = parcel.readValue(Int::class.java.classLoader) as? Int
        orderstatuscd = parcel.readValue(Int::class.java.classLoader) as? Int
        updateuid = parcel.readValue(Int::class.java.classLoader) as? Int
    }

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeValue(orderstatusid)
        parcel.writeValue(orderid)
        parcel.writeValue(orderstatuscd)
        parcel.writeValue(updateuid)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<OrderStatus> {
        override fun createFromParcel(parcel: Parcel): OrderStatus {
            return OrderStatus(parcel)
        }

        override fun newArray(size: Int): Array<OrderStatus?> {
            return arrayOfNulls(size)
        }
    }


}